### PR TITLE
cmds: add tp to vehicle id admin command

### PR DESCRIPTION
**Pull request by duboiss**

### CODE REVIEW

File: SemiRP/Utils/AdminUtils/AdminHelper.cs [hunk 1]
```diff
@@ -183,30 +183,47 @@ public static void VehicleDestroy(Player sender, int id = -1)
...
+            VehicleTp(sender, nearestVeh);
             return nearestVeh;
         }
 
+        public static Vehicle VehicleTpId(Player sender, int id)
+        {
+            if (!sender.AccountData.HavePerm("admin.cmds.vehicule.tpid"))
+                throw new Exception();
+
+            var vehicle = (Vehicle)Vehicle.Find(id);
```
Comment (V1ck3s) [inline SemiRP/Utils/AdminUtils/AdminHelper.cs:210]: No need to cast here, use Vehicle type directly
```suggestion
            Vehicle vehicle = Vehicle.Find(id);
```

**Review by V1ck3s — CHANGES_REQUESTED** [review]
Good, there is just one small thing to fix.

**Review by V1ck3s — APPROVED** [review]

File: SemiRP/Commands/AdminCommands.cs [hunk 2]
```diff
@@ -203,13 +203,27 @@ private static void Destroy(Player sender, int id = -1)
                 }
             }
 
-            [Command("tpnearest", "tpn")]
-            private static void TpTo(Player sender)
+            [Command("tpn", "tpnearest")]
```
Comment (Papawy) [inline SemiRP/Commands/AdminCommands.cs]: We should always but the full command name in first position.

File: SemiRP/PermissionList.cs [hunk 3]
```diff
@@ -31,7 +31,8 @@ public static class PermissionList
             "admin.cmds.vehicule.spawntmp",
             "admin.cmds.vehicule.create",
             "admin.cmds.vehicule.destroy",
-            "admin.cmds.vehicule.tp",
+            "admin.cmds.vehicule.tpn",
+            "admin.cmds.vehicule.tpid",
```
Comment (Papawy) [inline SemiRP/PermissionList.cs]: Just put `admin.cmds.vehicule.tp` not `tpid`.

**Review by Papawy — COMMENTED** [review]
There is some mistakes due to our coding standards for this codebase, but otherwise it is good.

**Comment by Papawy** [comment]
Squash your commits @DuboisS.

**Pull request merged**